source/docs/installation.md

```diff
@@ -0,0 +1,59 @@
+title: Installation
+---
+
+## Prerequisites
+
+In order to make Embark work on our machine, we need to have some tools installed first. Make sure you have the following ready and in the correct version:
+
+- [Node](#Node)
+- [IPFS](#IPFS-Optional)
+- [Ethereum Client](#Ethereum-Client-Optional)
+
+Once done, go ahead and [install Embark](#Installing-Embark).
+
+### Node
+
+Please install [Node.js](http://nodejs.org/) in version 8.11.3 LTS or higher.
+
+{% note info Quick tip: %}
+**We recommend installing Node using the [Node Version Manager](https://github.com/creationix/nvm/blob/master/README.md).**  This is because it makes it very easy to install different versions of Node in isolated environments that don't require users to [change their permissions](https://docs.npmjs.com/getting-started/fixing-npm-permissions) when installing packages. Find instructions on how to install NVM [here](https://github.com/creationix/nvm/blob/master/README.md#install-script).
+
+Once that is done, you can install and select a specific Node version using the `nvm` command like this:
+
+<pre><code class="shell">$ nvm install 8.11
+$ nvm use 8.11
+</code></pre>
+
+{% endnote %}
+
+### IPFS (Optional)
+
+IPFS can be used to distribute your application's content on the decentralized IPFS nodes. This can be skipped in case this isn't planned, however we do recommend it. Checkout IPFS' [installation guide](https://docs.ipfs.io/introduction/install/) to learn how to install IPFS on your local machine.
+
+To verify that the installation was successful, simply run the following command:
+
+<pre><code class="shell">$ ipfs --version
+</code></pre>
+
+### Ethereum Client (Optional)
+
+Embark can spin up an Ethereum node for you. To make this happen, an Ethereum client has to be installed on your machine as well. Embark already comes with [Ganache CLI](https://truffleframework.com/ganache), a blockchain node emulator, which is accessible via [Embark's simulator](embark_commands.html#simulator) command.
+
+In case you want to run a real node, [geth](https://geth.ethereum.org/) is a pretty good one. Check out the [installation guide](https://ethereum.github.io/go-ethereum/install/) for your platform and verify your installation with:
+
+<pre><code class="shell">$ geth --version</code></pre>
+
+## Installing Embark
+
+Alright, let's install Embark so we can build our first application. As mentioned earlier, if anything is unclear or you run into problems, make sure to reach out to us on our dedicated channels or [submit an issue on  GitHub](https://github.com/embark-framework/embark/issues).
+
+We can install Embark using the Node Package Manager (no worries, that one comes with Node), like this:
+
+<pre><code class="shell">$ npm -g install embark</code></pre>
+
+After that, `embark` should be available as a global command in your terminal of choice. Let's verify this by running the following command:
+
+<pre><code class="shell">$ embark --version</code></pre>
```

Comment:
Can we include the output from the tool? I want to see what to look for when I run this command.

Reply:
It's actually `embark version`

Reply:
uhm.. `embark --version` does the trick though

Reply:
Both do the same, but if you prefer going with `version`, I can update this.

Reply:
Really? Was this added recently? Anyway, TIL
You can keep it as is then.